core/src/main/java/org/apache/carbondata/core/cache/CarbonLRUCache.java

```diff
@@ -43,10 +45,11 @@
   private static final Logger LOGGER =
       LogServiceFactory.getLogService(CarbonLRUCache.class.getName());
   /**
-   * Map that will contain key as table unique name and value as cache Holder
+   * Guava cache that holds key as table unique name and value as cache Holder
    * object
    */
-  private Map<String, Cacheable> lruCacheMap;
+  private Cache<String, Cacheable> lruCache;
```

Comment:
since now least recently used cache is changed to time based, does this variable name still makes sense? shall we give a better one to depict the time based cache implementation?